jcl/src/openj9.dtfjview/share/classes/module-info.java

```diff
@@ -29,7 +29,7 @@
   requires openj9.dtfj;
   requires java.logging;
 
-  exports com.ibm.jvm.dtfjview.commands to openj9.dtfj;
+  exports com.ibm.jvm.dtfjview.commands;
```

Comment:
This seems reasonable, but I don't see any use in openj9.dtfj (as should be the case, otherwise there's a dependency cycle involving the two modules), so I don't know why it was added in the first place.